React/CoreModules/RCTLogBoxView.h

```diff
@@ -11,27 +11,25 @@
 
 #if !TARGET_OS_OSX // TODO(macOS GH#774)
 
-@interface RCTLogBoxWindow : UIWindow // TODO(macOS GH#774) Renamed from _view to _window
+@interface RCTLogBoxView : UIWindow
```

Comment:
Given that in core this is a UIWindow, would it make more sense to upstream our rename instead? This topic came up in previous merges and I wasn't sure if it made sense to.

Reply:
I would just KISS ;-) and optimize for 'easy merge'.

There are 5 more commits we have to merge and all cause more pain due to renaming `RCTLogBoxView` to `RCTLogBoxWindow` > https://github.com/facebook/react-native/commits/main/React/CoreModules/RCTLogBoxView.mm

Now:
- Each merge requires to remap `_view` to `_windows` AND
- Factor in the `#ifdef` macOS `#else` ... `#endif` logic

If we would rename it, we also have to rename the file (and the imports) and we would only do it in 0.71 - not solving the immediate problem here.

The most painful resolution would be:
- rename back here
- merge till 0.71
- (if we want, rename back in core in 0.71)

Reply:
Fair enough. I logged a task internally for me to followup on this.

Reply:
You can still rename it in react-native@main branch.
I just would not do it, before all current changes are merged

Reply:
Yep! If anything I'll do it after the 0.71 merge completes, when theoretically we are closer to RN core and it's easier to cherry pick changes across.

Reply:
But then RN core might already be at 0.72 or 0.73 ... a race which never can be won ... ;-)

Reply:
So better create a PR now ;-)

Reply:
https://github.com/facebook/react-native/pull/35358

Reply:
Now or never ;-)